hazelcast/src/main/java/com/hazelcast/config/GroupConfig.java

```diff
@@ -102,7 +102,10 @@ public String getPassword() {
      * @param password the password to set for the group
      * @return the updated GroupConfig
      * @throws IllegalArgumentException if password is {@code null}
+     * @deprecated since 3.11, password check is removed.
```

Comment:
should you put the same deprecation for getPassword as well?

Reply:
added